tests/integrationv2/README.md

```diff
@@ -0,0 +1,100 @@
+# Quick start
+
+You need to install Tox to help setup your environment. Once tox is installed, run it to setup your
+local environment. Also set your path to access the tox artifacts.
+
+```
+$ tox
+py38 create: /home/ubuntu/development/s2n/tests/integrationv2/.tox/py38
+py38 installdeps: pep8, pytest==5.3.5, pytest-xdist
+py38 installed: apipkg==1.5,attrs==19.3.0,execnet==1.7.1,importlib-metadata==1.6.0,more-itertools==8.2.0,packaging==20.3,pep8==1.7.1,pkg-resources==0.0.0,pluggy==0.13.1,py==1.8.1,pyparsing==2.4.7,pytest==5.3.5,pytest-forked==1.1.3,pytest-xdist==1.32.0,six==1.14.0,wcwidth==0.1.9,zipp==3.1.0
+
+$ export PATH=.tox/py38/bin:$PATH
+```
+
+You can run the entire test suite using `sudo make`. You need to use `sudo` if you plan on running the dynamic
+record tests.
+
+If you don't want to run everything, you can run individual tests like this:
+
+```
+$ pytest -rpfsq test_happy_path.py::test_s2n_server_happy_path
+```
+
+# A toy example
+
+The happy path test combines thousands of parameters, and has to validate that the
+combinations match. This makes it difficult to use as a toy example. Below is a
```

Comment:
I'd eliminate "This makes it difficult to use as a toy example." as it doesn't actually add value to what you are trying to get across

Reply:
Makes sense, I'll remove.